setup/docker/01_setup_enketo.bash

```diff
@@ -17,23 +17,33 @@ SHA1SUM_CONFIG_FILE_PATH="${CHECKSUM_DIR_PATH}/${CONFIG_FILE}.sha1"
 LAST_BUILD_COMMIT_FILE="last_build_commit.txt"
 LAST_BUILD_COMMIT_FILE_PATH="${CHECKSUM_DIR_PATH}/${LAST_BUILD_COMMIT_FILE}"
 
+RUN_GRUNT=0
 # Compare config version
-set +e  # Do not exit on `sha1sum` or `cat` error
-sha1sum -c "${SHA1SUM_CONFIG_FILE_PATH}"
-RUN_GRUNT=$([[ $? -eq 0 ]] && echo 0 || echo 1)
+sha1sum -c "${SHA1SUM_CONFIG_FILE_PATH}" || RUN_GRUNT=1
 
 # Compare commit version
 CURRENT_COMMIT=$(git rev-parse HEAD) # Get current commit
-LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH})
+LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH}) || echo "-"
 RUN_GRUNT=$([[ "$RUN_GRUNT" == 0 && "$CURRENT_COMMIT" == "$LAST_BUILD_COMMIT" ]] && echo 0 || echo 1)
 
-set -e  # Restore exit mode
+# Ensure build files exist.
+# Can't test only on the parent folder because it's created by docker
+NOT_EMPTY_JS_PATH="public/js/build/.not-empty"
+NOT_EMPTY_CSS_PATH="public/css/.not-empty"
+NOT_EMPTY_LOCALES_PATH="locales/build/.not-empty"
+
+RUN_GRUNT=$([[ "$RUN_GRUNT" == 0 && -f "$NOT_EMPTY_JS_PATH" && -f "$NOT_EMPTY_CSS_PATH" && -f "$NOT_EMPTY_LOCALES_PATH" ]] && echo 0 || echo 1)
```

Comment:
This should work just fine, but I'd gently suggest some line breaks to make it easier on the eyes :) Maybe:
```bash
RUN_GRUNT=$([[
    "$RUN_GRUNT" == 0 &&
    -f "$NOT_EMPTY_JS_PATH" &&
    -f "$NOT_EMPTY_CSS_PATH" &&
    -f "$NOT_EMPTY_LOCALES_PATH"
]] && echo 0 || echo 1)
```

Reply:
I usually break lines at 120 characters but I have to admit I abused on this one ;-)